great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -246,7 +247,7 @@ def get_expectation_suite(
         )
         if not self.expectations_store.has_key(key):  # noqa: W601
             raise ge_exceptions.DataContextError(
-                f"expectation_suite {expectation_suite_name} not found"
+                f"expectation_suite with id {ge_cloud_id} not found"
```

Comment:
`has_key` checks always use ID while `name in self.list_expectation_suite_names()` should reference name